script/mbx

```diff
@@ -544,6 +551,9 @@ def webwork_to_xml(xslt_exec, ptx_xsl_dir, xml_source, server_params, dest_dir):
         elif bad_xml:
             print(bad_xml_msg.format(problem_identifier, seed[problem],'  Use -a to halt with returned content'))
             static[problem] = "<static failure='xml'>\n<statement>\n  <p>\n    " + bad_xml_msg.format(problem_identifier, seed[problem],'    Use -a to halt with returned content') + "  </p>\n</statement>\n</static>"
+        elif no_statement:
+            print(no_statement_msg.format(problem_identifier, seed[problem],'  Use -a to halt with returned content'))
+            static[problem] = "<static failure='xml'>\n<statement>\n  <p>\n    " + no_statement_msg.format(problem_identifier, seed[problem],'    Use -a to halt with returned content') + "  </p>\n</statement>\n</static>"
```

Comment:
Should this be @failure="no_statement" here? I was out of the office yesterday and haven't had a chance to test, but thought I'd look this over this morning and get to testing a bit later today.

Reply:
Good catch!

Reply:
I made it `failure='statement'` to keep it one word like the others. They don't all read well as "It failed because of _____" anyway.  So far, I believe there is nothing that uses these `@failure` attributes, but it seemed like they might come in handy some day.